lxd/api.go

```diff
@@ -78,6 +78,17 @@ func restServer(d *Daemon) *http.Server {
 		})
 	}
 
+	// Serving the LXD documentation.
+	documentationPath := os.Getenv("LXD_DOCUMENTATION")
```

Comment:
Can you see if the LXD_UI env var is documented, as we should probably document this in same place.